File: 07_기본수학_1/Problem_08.cpp
/* 07단계 [ 기본수학 1 ] */
// 제목 : 큰 수 A+B
// 문제 번호 : 10757

#include <iostream>
using namespace std;

int main(void)
{
    string A, B, C;
    int size;
    cin >> A >> B;
    size = max(A.length(), B.length());
    int sum[size + 1];
    for (int i = 0; i <= size; i++)
        sum[i] = 0;
    for (int i = 0; i < A.length(); i++)
        sum[i] += int(A[A.length() - i - 1]) - 48;
    for (int i = 0; i < B.length(); i++)
        sum[i] += int(B[B.length() - i - 1]) - 48;
    for (int i = 0; i < size; i++)
        if (sum[i] > 9)
        {
            sum[i] -= 10;
            sum[i + 1] += 1;
        }
    for (int i = 0; i <= size; i++)
        C += to_string(sum[i]);
    for (int i = C[C.length() - 1] == '0'; i < C.length(); i++)
        cout << C[C.length() - 1 - i];
}

File: Python/06_문자열/Problem_08.py
/* 06단계 [ 문자열 ] */
// 제목 : 다이얼
// 문제 번호 : 5622

#include <iostream>
using namespace std;

int main(void)
{
    int arr[26] = {3, 3, 3, 4, 4, 4, 5, 5, 5, 6, 6, 6, 7, 7, 7, 8, 8, 8, 8, 9, 9, 9, 10, 10, 10, 10};
    int sum = 0;
    string s;
    cin >> s;
    for (int i = 0; i < s.length(); i++)
        sum += arr[int(s[i]) - 65];
    cout << sum << endl;
}